pkg/webreg/webreg.go

```diff
@@ -403,6 +404,231 @@ const templateDefinitions = `
 {{ end }}
 `
 
+const optionalOperatorOverviewPage = `<h2 id="title"><a href="#title">Testing Operators Built With The Operator SDK and Deployed Through OLM</a></h2>
+
+<p>
+<code>ci-operator</code> supports building, deploying and testing operator bundles,
+whether the operator repository uses the Operator SDK or not.
+This document outlines how to configure <code>ci-operator</code> to build bundle
+and index images and use those in end-to-end tests.
+</p>
+
+<p>
+Consult the <code>ci-operator</code> <a href="/help/ci-operator">overview</a> and
+the step environment <a href="/help">reference</a> for detailed descriptions of the
+broader test infrastructure that an operator test is defined in.
+</p>
+
+<h3 id="operator-artifacts"><a href="#operator-artifacts">Building Artifacts for OLM Operators</a></h3>
+
+<p>
+Multiple different images are involved in the process of installation and testing
+candidate versions of OLM-delivered operators: operands, operator, bundle and index.
```

Comment:
This would read better with one of the following to have parallel construction:

```
operators: operand, operator, bundle and index images.
```

```
operators: operands, operators, bundles and indicies.
```